apps/activejobs/app/assets/stylesheets/jobs.scss

```diff
@@ -25,3 +25,23 @@ tr.shown td.details-control {
   right: 0;
   height: auto;
 }
+
+.action-btn {
+  margin: 1px;
+}
+
+.action-btn .action-btn-tooltip {
+  visibility: hidden;
+  width: 120px;
+  background-color: black;
+  color: white;
+  text-align: center;
+  border-radius: 6px;
+  padding: 5px 0;
+  position: absolute;
+}
+
+.action-btn:hover .action-btn-tooltip {
+  visibility: visible;
+}
```

Comment:
You should use Bootstrap tooltip and instead of implementing your own. Copy the implementation of the Job Composer's delete button: https://getbootstrap.com/docs/3.4/javascript/#tooltips

Basically you should just have to add these to the link attrs: `data-toggle="tooltip"` and `title="Delete Job"`